pandas/plotting/_core.py

```diff
@@ -1935,52 +1935,134 @@ def plot_series(data, kind='line', ax=None,                    # Series unique
+    quartile values of the data, with a line at the median (Q2).
+    The whiskers extend from the edges of box to show the range of the data.
+    Flier points (outliers) are those past the end of the whiskers.
+    The position of the whiskers is set by default to 1.5 IQR (`whis=1.5``)
+    from the edge of the box.
+
+    For further details see
+    Wikipedia's entry for `boxplot <https://en.wikipedia.org/wiki/Box_plot/>`_.
 
     Parameters
     ----------
-    data : the pandas object holding the data
     column : column name or list of names, or vector
-        Can be any valid input to groupby
+        Can be any valid input to groupby.
     by : string or sequence
-        Column in the DataFrame to group by
-    ax : Matplotlib axes object, optional
+        Column in the DataFrame to groupby.
+    ax :  Matplotlib axes object, (default `None`)
+        The matplotlib axes to be used by boxplot.
     fontsize : int or string
+        The font-size used by matplotlib.
     rot : label rotation angle
+        The rotation angle of labels.
+    grid : boolean( default `True`)
+        Setting this to True will show the grid.
     figsize : A tuple (width, height) in inches
-    grid : Setting this to True will show the grid
+        The size of the figure to create in inches by default.
     layout : tuple (optional)
-        (rows, columns) for the layout of the plot
+        Tuple (rows, columns) used for the layout of the plot.
     return_type : {None, 'axes', 'dict', 'both'}, default None
         The kind of object to return. The default is ``axes``
         'axes' returns the matplotlib axes the boxplot is drawn on;
         'dict' returns a dictionary whose values are the matplotlib
         Lines of the boxplot;
         'both' returns a namedtuple with the axes and dict.
-
         When grouping with ``by``, a Series mapping columns to ``return_type``
         is returned, unless ``return_type`` is None, in which case a NumPy
         array of axes is returned with the same shape as ``layout``.
         See the prose documentation for more.
-
-    `**kwds` : Keyword Arguments
+    kwds : Keyword Arguments (optional)
         All other plotting keyword arguments to be passed to
-        matplotlib's boxplot function
+        matplotlib's function.
 
     Returns
     -------
     lines : dict
     ax : matplotlib Axes
-    (ax, lines): namedtuple
+        (ax, lines): namedtuple
+
+    See Also
+    --------
+    matplotlib.pyplot.boxplot: Make a box and whisker plot.
 
     Notes
     -----
     Use ``return_type='dict'`` when you want to tweak the appearance
     of the lines after plotting. In this case a dict containing the Lines
     making up the boxes, caps, fliers, medians, and whiskers is returned.
-    """
 
+    Examples
+    --------
+    .. plot::
+        :context: close-figs
+
+        >>> np.random.seed(1234)
+
+        >>> df = pd.DataFrame({
+        ...     u'stratifying_var': np.random.uniform(0, 100, 20),
```

Comment:
The docs guide advises to try avoid using random data. In this case I believe it's justifiable because there is a relation between random distributions and boxplots and this can be shown in examples.

However please explain in plain English that you are creating a dataframe with points following a uniform and a normal distribution, respectively, in the different columns. You can also probably find two variables that are naturally uniform and normal, respectively.

Reply:
@Lkxz See this ^^ . @datapythonista what do you think? Is it OK to use random data with a seed sometimes in examples if it makes sense?

Reply:
I agree, feel free to update the documentation too.

Reply:
BTW do we want to use the unicode mark `u''` at all in documentation examples? I believe we should always write strings without `u` or `b` unless necessary. This way it will be a bytestring in Python 2 and a text (unicode) in Python 3.